qiskit_experiments/library/hamiltonian/heat_zx.py

```diff
@@ -0,0 +1,315 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+HEAT experiments for ZX Hamiltonian.
+"""
+
+from typing import Tuple, Optional
+
+import numpy as np
+from qiskit.circuit import QuantumCircuit, Gate
+from qiskit.providers import Backend
+
+from qiskit_experiments.framework import Options, composite
+from .heat_analysis import HeatAnalysis
+from .heat_base import HeatElement
+
+
+@composite.sync_experiment_options
+@composite.sync_transpile_options
+class ZXHeat(composite.BatchExperiment):
+    r"""HEAT experiment for the ZX-type entangler.
+
+    # section: overview
+        This experiment is designed to amplify the error contained in the
+        ZX-type generator, a typical Hamiltonian implemented
+        by a cross-resonance drive, which is typically used to create a CNOT gate.
```

Comment:
This is a bit nit-picky.
```suggestion
        This experiment is designed to amplify the errors contained in, e.g., cross-resonance
        Hamiltonians. These Hamiltonians are typically used to create ZX rotations which
        can create a CNOT gate. The cross-resonance Hamiltonian typically contains
        other generators, such as IZ and IX, whose strength can be determined with the
        `ZXHeat` experiment.
```
I suggest to reformulate this since the errors are not contained in the ZX but rather the ZX is contained in a Hamiltonian which has errors, i.e. the non-ZX terms.

Reply:
No ZX can have error from the expected rotation angle. It just says primary term of Hamiltonian is ZX. This dominates the initialization circuit. Usually we need to prepare the circuit in the eigenstate of primary generator.